#> cat obstacle_info.msg

int32 id
bool isobstacle